### DISCUSSION TITLE
Users cannot select their seller profile when doing oauth authorization

### DISCUSSION THREAD

**Discussion by mlepine** [amzn/selling-partner-api-models#603] [Ideas]
## I'm submitting a...

<pre><code>[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ x] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

We have many clients who have multiple amazon seller central profiles. Since the oauth process doesn't prompt for the user to select the profile and instead selects the current profile, they often authorizes access to the wrong account and have to reauthorized. There is also no way for us to show the user which profile they had authorized so we are unable to detect the error until data is pulled into the application.

**Comment by stefnats** [comment]
Hey @mlepine i don't have a solution, but a suggestion.

Maybe you could call [getMarketplaceParticipations](https://developer-docs.amazon.com/sp-api/docs/sellers-api-v1-reference#getmarketplaceparticipations) to display at least the seller id / merchant id and the countries that the seller is participating. 

You could then display a link to double check before all gets pulled in.
